charts/rancher-monitoring/100.0.0+up16.6.0/Chart.yaml

```diff
@@ -94,7 +96,7 @@ keywords:
 - prometheus
 - kube-prometheus
 - monitoring
-kubeVersion: '>=1.16.0-0'
+kubeVersion: '>= 1.16.0-0 < 1.22.0-0'
```

Comment:
should not be narrowing the scope of the kubeVersion after-the-fact, should leave as-is